frontend/templates/troubleshooting/index.tsx

```diff
@@ -130,126 +110,73 @@ const Wiki: NextPageWithLayout<{
    return (
       <>
          <GoogleNoScript />
+         <TagManager />
+         <Metadata
+            metaDescription={metaDescription}
+            metaKeywords={metaKeywords}
+            canonicalUrl={canonicalUrl}
+            title={title}
+         />
+         <HreflangUrls urls={wikiData.hreflangUrls} />
+
          <NavBar
             editUrl={wikiData.editUrl}
             historyUrl={wikiData.historyUrl}
             deviceGuideUrl={wikiData.deviceGuideUrl}
             devicePartsUrl={wikiData.devicePartsUrl}
             breadcrumbs={wikiData.breadcrumbs}
          />
+
          <TOCContextProvider defaultItems={tocItems}>
-            <Flex>
+            <Box
+               className="layout-grid"
+               display="grid"
+               sx={{
+                  gridTemplateColumns: {
+                     base: '[toc] none [wrapper] 1fr',
+                     lg: '[toc] auto [wrapper] 1fr',
+                  },
+               }}
+               ref={contentContainerRef}
+            >
                <TOC
+                  className="summary"
                   contentRef={contentContainerRef}
-                  flexShrink={{ lg: 0 }}
-                  flexGrow={1}
                   borderRight={{ lg: '1px solid' }}
                   borderColor={{ lg: 'gray.300' }}
-                  maxWidth={{
-                     base: 'calc(100% + 2 * var(--chakra-space-4))',
-                     lg: '240px',
-                  }}
-                  marginLeft={-4}
-                  marginRight={{ base: -4, lg: 0 }}
-                  listItemProps={{
-                     paddingLeft: { lg: 4 },
-                  }}
+                  maxWidth={{ base: 'calc(100% + 2 * var(--chakra-space-4))' }}
+                  listItemProps={{ paddingLeft: { lg: 4 } }}
+                  gridArea="toc"
+                  width="220px"
```

Comment:
Minor: I see this changed from 240 -> 220. I suspect you got clarity from the figma here?

Reply:
Yep:
<img width="1315" alt="Screenshot 2023-09-25 at 3 30 36 PM" src="https://github.com/iFixit/react-commerce/assets/1634505/0effaf9c-8bb7-4335-8d4f-3ebb4df848f9">